README.md

```diff
@@ -116,10 +114,38 @@ The following prerequisites are required:
 - Maven 3.6.3 or higher
 - docker 20.10.12 or higher
 - docker-compose 1.29.2 or higher
+- By default, CoatRack will use GitHub as the identity manager;
 
 With your terminal go into the `coatrack/docker` directory where you will find the script `build-and-deploy-images-locally.sh` which builds CoatRack from source, creates container images on your local machine and runs CoatRack locally using these images. This is especially useful for developers who did changes to the source code and want to locally test the impact of these changes in a realistic setup.
 
-
+## Use FIWARE Keyrock for Identity Management instead of Github
+By default, CoatRack uses Github for identity management and login. As an alternative, you can setup CoatRack to use KeyRock IdM, which is the FIWARE enabler for Identity Management. In this case you have to setup KeyRock first and then setup CoatRack.
+Moreover, the example shown below will consider that you deployed CoatRack in localhost:8080, and that Keyrock was deployed via Docker with the version 7.8.1 (fiware/idm:7.8.1).
+
+Based on this, in order to make CoatRack work with the FIWARE Keyrock, you will need to:
+
+- First, pull and deploy the containers of keyrock via docker (as the Keyrock itself needs to be running in your local machine). To do this, just follow
+the tutorial in https://fiware-idm.readthedocs.io/en/latest/getting_started/index.html;
+- Please check that Keyrock is running correctly in your local machine;
+- If everything is running well, then we need to start by register CoatRack as application in Keyrock, to do this:
+  - Login or Create a user account (you can use the admin@test.com, which is the default admin account);
```

Comment:
```suggestion
 - Create a new user account or log in with the account `admin@test.com`, which is created as the administrator account by default.
```

Reply:
I applied now this suggestion, so consider this done